libs/redis/files/redis.init

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+START=95
+STOP=10
+
+REDIS_BIN="/usr/bin/redis-server"
+REDIS_CONFIG="/etc/redis.conf"
+REDIS_PID="/var/run/redis.pid"
+REDIS_DATA="/var/lib/redis"
+
+start_service() {
+	mkdir -p "$REDIS_DATA"
```

Comment:
How about moving this into the `package/xxx/install`?

Reply:
I think that it will not work since /var/lib/redis will be in a ramdisk, or I'm I mistaken?

Reply:
You are right.  The the directory can be absent after reboot

My concern is that the initscript should create directory in accordance with the configuration file.  Otherwise the hardcoded logic will be unrelevant on configuration change.

Reply:
I understand, but I think that it has no clean solution because if a user decides to change the default directory he/she has to set it appropriately and create it because it can be set in included configuration for example...

Reply:
I see.  It's indeed about balance and tradeoff ;)